trajopt_sco/src/solver_utils.cpp

```diff
@@ -90,15 +90,30 @@ void exprToEigen(const AffExprVector& expr_vec,
   sparse_matrix.resize(static_cast<long int>(expr_vec.size()), n_vars);
   sparse_matrix.reserve(static_cast<long int>(expr_vec.size()) * n_vars);
 
-  for (long int i = 0; i < static_cast<long int>(expr_vec.size()); ++i)
+  using T = Eigen::Triplet<double>;
+  std::vector<T, Eigen::aligned_allocator<T>> triplets;
```

Comment:
I recommend adding this to typedefs.hpp.
```
template <typename T>
using AlignedVector = std::vector<T, Eigen::aligned_allocator<T>>;
```
then replace this with
`AlignedVector<Eigen::Triplet<double>> triplets;`
 
Also the function below could be updated `tripletsToEigen`

Reply:
I'll handle this in a future PR